tests/unit/bin/update-ips/resources/maximal-run-full-diff.out

```diff
@@ -1,63 +1,61 @@
 [1m--- common-config.yaml[0m
 [1m+++ expected[0m
-[36m@@ -4,3 +4,8 @@[0m
- global:
-   opsDomain: ops.dev-ck8s.com
-   baseDomain: dev-ck8s.com
-[32m+networkPolicies:[0m
-[32m+  global:[0m
-[32m+    objectStorage:[0m
-[32m+      ips:[0m
+[36m@@ -22,7 +22,7 @@[0m
+   global:
+     objectStorage:
+       ips:
+[31m-        - set-me[0m
```

Comment:
Why has the result changed?

Reply:
Because the basis for the test now uses a config generated by init instead of a completely empty one. And that sets `set-me`s for networkpolices prompting for the configuration.

Also the base config changed as I worked to make it as much reusable as possible, and change `"true"`s into boolean `true`s.